core/tee/tee_svc_storage.c

```diff
@@ -43,15 +43,13 @@ static const struct tee_file_operations *file_ops(uint32_t storage_id)
 
 	switch (storage_id) {
 	case TEE_STORAGE_PRIVATE:
-#ifdef CFG_REE_FS
+#if defined(CFG_REE_FS)
```

Comment:
I would add the comment you gave in the commit message, that is `The value TEE_STORAGE_PRIVATE will select the REE FS if available, otherwise RPMB`